src/Fetcher.ts

```diff
@@ -35,8 +35,10 @@ export default abstract class Fetcher {
   ): Promise<ImageDimensions | undefined>;
 
   /**
-   * For the time being, we assume that encryption.xml is located
-   * as a sibling of container.xml
+   * Get the file located inside the META-INF folder
    */
-  abstract getEncryptionPath(containerXmlPath: string): string;
+  abstract getResourcePathByFileName(
```

Comment:
This might need a better name. The goal of the function is to search for the provided file in the META-INF folder. We are using this function to search the encryption.xml and the *-display-options.xml

Reply:
Maybe `searchForFileInMetaInf` haha? Very specific, but maybe that's best?

Reply:
Ok actually, it looks like this is not searching for the file in the concrete implementation. It's just appending the filename to `/META-INF`. So I think maybe we could rename it `createPathToFileInMetaInf` or something?